### PR TITLE
User/host details flyouts + Asset criticality

### PR DESCRIPTION
**Pull request by natasha-moore-elastic**
Contributes to the following issues:

- https://github.com/elastic/security-docs/issues/4459 and https://github.com/elastic/security-docs/issues/4576 by documenting the new expandable user and host details flyouts
- https://github.com/elastic/security-docs/issues/4697 by updating the information on how risk score is calculated
- https://github.com/elastic/security-docs/issues/4841 by documenting the asset criticality feature

Twin serverless PR: https://github.com/elastic/staging-serverless-security-docs/pull/280

### Previews:

New page:
* [Asset criticality](https://security-docs_bk_4939.docs-preview.app.elstc.co/guide/en/security/master/asset-criticality.html)

Updated pages:
* [View and analyze risk score data](https://security-docs_bk_4939.docs-preview.app.elstc.co/guide/en/security/master/analyze-risk-score-data.html)
* [Entity risk scoring](https://security-docs_bk_4939.docs-preview.app.elstc.co/guide/en/security/master/entity-risk-scoring.html)
* [Turn on the risk scoring engine](https://security-docs_bk_4939.docs-preview.app.elstc.co/guide/en/security/master/turn-on-risk-engine.html)
* [Manage detection alerts](https://security-docs_bk_4939.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-manage.html)
* [Hosts page](https://security-docs_bk_4939.docs-preview.app.elstc.co/guide/en/security/master/hosts-overview.html)
* [Users page](https://security-docs_bk_4939.docs-preview.app.elstc.co/guide/en/security/master/users-page.html)
* [Configure advanced settings](https://security-docs_bk_4939.docs-preview.app.elstc.co/guide/en/security/master/advanced-settings.html)
* [Entity risk scoring prerequisites](https://security-docs_bk_4939.docs-preview.app.elstc.co/guide/en/security/master/ers-requirements.html) - has ESS-specific updates

### CODE REVIEW

**Review by joepeeples — APPROVED** [review]
LGTM! 💪

File: docs/getting-started/ers-req.asciidoc [hunk 1]
```diff
@@ -35,8 +38,25 @@ Follow these guidelines to ensure clusters have adequate memory to handle data v
...
+== Asset criticality
+
+To use the asset criticality feature, turn on the `securitySolution:enableAssetCriticality` <<enable-asset-criticality, advanced setting>>.
+
+[discrete]
+=== Privileges
+
+* To view an entity's asset criticality, you need the `read` privilege for the `.asset-criticality.asset-criticality-<space-id>` index.
+
+* To view, assign, or change an entity's asset criticality, you need the `read` and `write` privileges for the `.asset-criticality.asset-criticality-<space-id>` index.
```
Comment (natasha-moore-elastic) [inline docs/getting-started/ers-req.asciidoc:57]: @elastic/security-entity-analytics, are there any cluster or Kibana privileges that we need to list here?

Comment (jmikell821) [reply docs/getting-started/ers-req.asciidoc:57]: cc: @jaredburgettelastic @SourinPaul @oatkiller see above please ⬆️

**Review by jaredburgettelastic — APPROVED** [review]

File: docs/getting-started/ers-req.asciidoc (hunk 1 above)
Comment (jaredburgettelastic) [reply docs/getting-started/ers-req.asciidoc:57]: No additional cluster or Kibana privileges are required, only the index-level privileges you have described above.

**Pull request merged**